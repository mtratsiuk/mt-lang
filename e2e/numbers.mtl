(print 1)
(print 1_000)
(print -1_000)
(print 1.2)
(print .2)
(print 1.)
(print 1_000_000.2_222)
