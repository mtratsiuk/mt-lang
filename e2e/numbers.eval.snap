1
1000
-1000
1.2
0.2
1
1000000.2222
